addons/mail/static/src/js/thread_widget.js

```diff
@@ -134,6 +134,7 @@ var ThreadWidget = Widget.extend({
 
         // copy so that reverse do not alter order in the thread object
         var messages = _.clone(thread.getMessages({ domain: options.domain || [] }));
+        this._messages = messages;
```

Comment:
Add a comment as @alexkuhn asked pointing to the problematic code

> Would also make a single commit and clearly put a link to the problematic code in website_livechat so that it's clear why we need _messages in the ThreadWidget even without snailmail being installed.

Reply:
Forget it. I see it in the commit

Reply:
Has it meaning to do that on each third module using this class (with also the possible side effects for not remembering that it should be put)? I think it's better to put it on the base class, isn't it?

Reply:
Well it is redefined in `snailmail` anyway, but it doesn't matter as this is obsolete code, I don't care as long as the "bug" is fixed.

Still have runbot issue, I started rebuild to see how it goes. Maybe a rebase would help if it's not yet resolved after.